src/OpenTelemetry.Exporter.Jaeger/JaegerTraceExporter.cs

```diff
@@ -29,10 +29,9 @@ public class JaegerTraceExporter : SpanExporter, IDisposable
         private readonly IJaegerUdpBatcher jaegerAgentUdpBatcher;
         private bool disposedValue = false; // To detect redundant dispose calls
 
-        public JaegerTraceExporter(JaegerExporterOptions options)
+        public JaegerTraceExporter(JaegerExporterOptions options, Resource libraryResource)
```

Comment:
resource is specific to Span (even the service resource) and should not be passed to exporter during construction.